code/src/App.js

```diff
@@ -1,9 +1,29 @@
-import React from 'react'
+import React, { useState } from 'react'
+
+import { StartSection } from './components/StartSection'
+import { Form } from './components/Form'
 
 export const App = () => {
+  const [displayForm, setDisplayForm] = useState(false)
+
+/* function that is invoked when button is clicked. Changes displayform value to true and displays the form*/ 
+  const handleStartClick = event => {
+  setDisplayForm(true)
+  }
+
   return (
-    <div>
-      Find me in src/app.js!
-    </div>
+    <>
+      <main className='main-content-container'>
+        {! displayForm ? (
+        <section className='intro-text-container'>
+          <StartSection onStartClick={handleStartClick}/>  
+        </section> )
+        : (
+        <section className='form-container'>
+          <Form/>
+        </section>
+        )}
+      </main>
+    </>
```

Comment:
You don't need to wrap this in a React.fragment since the `<main>` is wrapping it all